tools/link.py

```diff
@@ -72,7 +72,6 @@
     # wasm-ld doesn't support soname or other dynamic linking flags (yet).   Ignore them
     # in order to aid build systems that want to pass these flags.
     '-allow-shlib-undefined': False,
-    '-rpath': True,
```

Comment:
I wonder if we should also add this to `SUPPORTED_LINKER_FLAGS` above?

Reply:
Maybe not? @sbc100 it's not entirely clear to me from reading the code.

Reply:
Thanks. I added `-rpath` there.